package.json

```diff
@@ -1,3 +1,46 @@
 {
-  "note": "Workaround. Bundlewatch CLI needs package.json in root."
+  "private": true,
+  "author": "",
+  "license": "MIT",
+  "engines": {
+    "node": "16.16.0",
+    "npm": ">=8.0.0"
+  },
+  "packageManager": "npm@8.0.0",
+  "workspaces": [
+    "apps/*",
+    "frontend/admin",
+    "frontend/common",
+    "frontend/talentsearch",
+    "packages/*"
+  ],
+  "scripts": {
+    "build": "h2-build && turbo run build --filter='./apps/*' --filter='./frontend/*' --filter='!@gc-digital-talent/e2e'",
+    "build:all": "turbo run build",
+    "build:css": "h2-build",
+    "build:project": "turbo run build",
+    "codegen": "npm run codegen --workspaces --if-present",
+    "intl-extract": "npm run intl-extract --workspaces --if-present",
+    "intl-compile": "npm run intl-compile --workspaces --if-present",
+    "dev": "turbo run dev --filter='./apps/*' --filter='./frontend/*' --filter='!@gc-digital-talent/e2e'",
+    "dev:project": "turbo run dev",
+    "lint": "turbo run lint",
+    "storybook:project": "npm run storybook",
+    "storybook": "MERGE_STORYBOOKS=true npm run storybook --workspace=admin",
+    "build-storybook": "MERGE_STORYBOOKS=true npm run build-storybook --workspace=admin",
+    "test": "turbo run test",
+    "watch": "turbo run watch --filter=@gc-digital-talent/web",
```

Comment:
Is it still possible to run watch on one of the unconverted projects?
![image](https://user-images.githubusercontent.com/8978655/208942237-70f488fc-ead6-4f87-9c2f-752ff9824d42.png)

Reply:
okay, I made some changes to how watch commands work. Part of that was adding the following scripts which run in a specific project.

- `npm run watch:admin`
- `npm run watch:talentsearch`